modules/microshift-nw-topology.adoc

```diff
@@ -5,7 +5,11 @@
 :_mod-docs-content-type: CONCEPT
 [id="microshift-network-topology_{context}"]
 = Network topology
-OVN-Kubernetes provides an overlay-based networking implementation. This overlay includes an OVS-based implementation of Service and NetworkPolicy. The overlay network uses the Geneve (Generic Network Virtualization Encapsulation) tunnel protocol. The pod maximum transmission unit (MTU) for the Geneve tunnel is set to the default route MTU if it is not configured.
+
+[role="_abstract"]
+OVN-Kubernetes provides an overlay-based networking implementation. This overlay includes an OVS-based implementation of Service and NetworkPolicy.
```

Comment:
Should this be `a Service` or `Services` and then same for NetPol?

Reply:
Yeah I saw that, too. Wasn't sure. Let me check with @ShudiLi

Reply:
OK, yeah, those are resources, I see it the K docs. TY!